### ISSUE TITLE
Update the plugin to {N} 5.0

### ISSUE THREAD

**Issue by sis0k0** [NativeScript/nativescript-intl#18]
- Bump the supported {N} platforms versions to 5.0 - [link](https://github.com/NativeScript/nativescript-intl/blob/master/src/package.json#L5) 
- Remove the `import assignments` as they are not compatible with ESModules and may mislead people - [link](https://github.com/NativeScript/nativescript-intl/blob/5b789f5a1cb033b81f1483e2dd0b66116c27606e/src/nativescript-intl.ios.d.ts) and [link](https://github.com/NativeScript/nativescript-intl/blob/5b789f5a1cb033b81f1483e2dd0b66116c27606e/src/nativescript-intl.android.d.ts)
- Test and publish

**Comment by wendt88** [comment]
@sis0k0 can this repo maintained by [NativeScript Community](https://github.com/nativescript-community) to be able to merge at least the bug fixing PR`s?

**Comment by sis0k0** [comment]
Great idea, @wendt88! @NathanWalker, do you think that would be possible?

**Comment by NathanWalker** [comment]
Yes absolutely - I can see about adding it to main plug-ins repo soon.
